resources/setups/common/base_config.json

```diff
@@ -49,7 +49,18 @@
       "enabled": true
     },
     "dot1x": {
-      "enabled": true
+      "enabled": true,
+      "interface": "eth0",
```

Comment:
What's this for?  Seems wrong to be exposing this level of detail -- at what point specifically does this need to be set, and what other values might it have?

Reply:
So, the interface is kinda important for the eap_module. I added a way to estimate the default one that's used in the tests. This can be used as an override. I will change this for dot1x test integration PR